swingset/src/main/java/com/nqadmin/swingset/SSBaseComboBox.java

```diff
@@ -136,6 +136,24 @@
 		@Override
 		public void actionPerformed(final ActionEvent ae) {
 
+// ***********
+// @errael this block that you added is taking place in updateRowset(), which is called at the bottom of this method (actionPerformed() )
+// ***********
+/*
+// ER selection-pending code
+			removeRowSetListener();
+
+			M mapping = getSelectedMapping();
+
+			if (mapping == null) {
+				setBoundColumnText(null);
+				logger.debug(() -> String.format("%s: Setting to null.", getColumnForLog()));
+			} else {
+				setBoundColumnText(String.valueOf(mapping));
+				logger.debug(() -> String.format("%s: Setting to %s.",  getColumnForLog(), mapping));
+      			}
+*/			
+// BP gl-strict-contains code
```

Comment:
@errael Any concerns about returning if we're dealing with a combo navigator (below) and keeping the above logic in updateRowset() around line 900? If not, we can leave the commented block for now in case there are unforeseen side effects and clean up later.

updateRowset() also has some of your event-bus prep code.

Reply:
So all of this (block above highlighted in green (and commented out by me in GitHub editor) was re-introduced into actionPerformed(). Based on your email, there should just be one line edited in updateRowset(). Will fix.